### ISSUE TITLE
Installation with `go get` didn’t complete on Ubuntu18.04.

### ISSUE THREAD

**Issue by Tocyuki** [motemen/ghq#199]
Installation with `go get` didn’t complete on Ubuntu18.04.

My environments are below.

```
$ go version
go version go1.12.13 linux/amd64
$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.3 LTS
Release:        18.04
Codename:       bionic
```

This is how to install ghq.

```
$ go get github.com/motemen/ghq
# github.com/motemen/ghq
go/src/github.com/motemen/ghq/commands.go:26:14: cannot use cli.BoolFlag literal (type cli.BoolFlag) as type cli.Flag in array or slice literal:
        cli.BoolFlag does not implement cli.Flag (Apply method has pointer receiver)
go/src/github.com/motemen/ghq/commands.go:27:14: cannot use cli.BoolFlag literal (type cli.BoolFlag) as type cli.Flag in array or slice literal:
        cli.BoolFlag does not implement cli.Flag (Apply method has pointer receiver)
go/src/github.com/motemen/ghq/commands.go:28:14: cannot use cli.BoolFlag literal (type cli.BoolFlag) as type cli.Flag in array or slice literal:
        cli.BoolFlag does not implement cli.Flag (Apply method has pointer receiver)
go/src/github.com/motemen/ghq/commands.go:29:14: cannot use cli.BoolFlag literal (type cli.BoolFlag) as type cli.Flag in array or slice literal:
        cli.BoolFlag does not implement cli.Flag (Apply method has pointer receiver)
go/src/github.com/motemen/ghq/commands.go:30:16: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in array or slice literal:
        cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
go/src/github.com/motemen/ghq/commands.go:31:14: cannot use cli.BoolFlag literal (type cli.BoolFlag) as type cli.Flag in array or slice literal:
        cli.BoolFlag does not implement cli.Flag (Apply method has pointer receiver)
go/src/github.com/motemen/ghq/commands.go:59:15: cannot use cli.BoolFlag literal (type cli.BoolFlag) as type cli.Flag in array or slice literal:
        cli.BoolFlag does not implement cli.Flag (Apply method has pointer receiver)
go/src/github.com/motemen/ghq/commands.go:60:17: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in array or slice literal:
        cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
go/src/github.com/motemen/ghq/commands.go:61:15: cannot use cli.BoolFlag literal (type cli.BoolFlag) as type cli.Flag in array or slice literal:
        cli.BoolFlag does not implement cli.Flag (Apply method has pointer receiver)
go/src/github.com/motemen/ghq/commands.go:62:15: cannot use cli.BoolFlag literal (type cli.BoolFlag) as type cli.Flag in array or slice literal:
        cli.BoolFlag does not implement cli.Flag (Apply method has pointer receiver)
go/src/github.com/motemen/ghq/commands.go:62:15: too many errors
```

If you have any idea please tell me that.

**Comment by cumet04** [comment]
`GO111MODULE=on` works for me.

**Comment by Songmu** [comment]
This error seems to be due to the latest incompatible changes in urfave/cli. It does not occur when using version-locked Go Modules. I'll consider how to address this issue.

**Comment by Tocyuki** [comment]
@cumet04 What does the following mean?

>`GO111MODULE=on`

**Comment by Songmu** [comment]
The meaning of GO111MODULE is that if GO Mosules is enabled and `go get` is executed, it would succeed. Try following command. It is current option. This is a current solution for now.

```
$ GO111MODULE=on go get github.com/motemen/ghq
```

**Comment by Songmu** [comment]
It may be fixed at #201.

However, if you just want a ghq executable, we don't recommend "go get".

It is recommended that you download the stable executable from GitHub Releases.

https://github.com/motemen/ghq/releases

**Comment by Songmu** [comment]
Released as v0.12.9. Thanks for reporting!
